src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+spring.application.name=psc-extensions-api
+
+# Spring Actuator
+management.endpoints.enabled-by-default=${MANAGEMENT_ENDPOINTS_ENABLED_BY_DEFAULT:false}
+management.endpoint.health.enabled=${MANAGEMENT_ENDPOINT_HEALTH_ENABLED:true}
+management.endpoints.web.path-mapping.health=${MANAGEMENT_ENDPOINTS_WEB_PATH_MAPPING_HEALTH:healthcheck}
+management.endpoints.web.base-path=${MANAGEMENT_ENDPOINTS_WEB_BASE_PATH:/psc-extensions}
+
```

Comment:
You may need to include `management.endpoints.web.exposure.include` as well so you can specify the endpoints you are enabling